src/test/UniswapV3SplitRouteCallee.t.sol

```diff
@@ -0,0 +1,359 @@
+            hevm.store(token, keccak256(abi.encode(uint256(i), address(this))), bytes32(amount));
+            if (GemAbstract(token).balanceOf(address(this)) == amount) {
+                // Found it
+                return;
+            } else {
+                // Keep going after restoring the original value
+                hevm.store(token, keccak256(abi.encode(uint256(i), address(this))), prevValue);
+            }
+        }
+    }
+
+    address linkAddr;
+    address daiAddr;
+    address vatAddr;
+    address linkJoinAddr;
+    address daiJoinAddr;
+    address dogAddr;
+    address jugAddr;
+    address linkClipAddr;
+    address linkPipAddr;
+
+    Hevm hevm;
+    GemAbstract     link;
+    VatAbstract     vat;
+    DaiAbstract     dai;
+    GemJoinAbstract linkJoin;
+    DogAbstract     dog;
+    JugAbstract     jug;
+    ClipAbstract    linkClip;
+
+    OsmAbstract linkPip;
+
+    function setAddresses() private {
+        ChainlogHelper helper = new ChainlogHelper();
+        ChainlogAbstract chainLog = helper.ABSTRACT();
+        linkAddr = chainLog.getAddress('LINK');
+        vatAddr = chainLog.getAddress('MCD_VAT');
+        daiAddr = chainLog.getAddress('MCD_DAI');
+        linkJoinAddr = chainLog.getAddress('MCD_JOIN_LINK_A');
+        daiJoinAddr = chainLog.getAddress('MCD_JOIN_DAI');
+        dogAddr = chainLog.getAddress('MCD_DOG');
+        jugAddr = chainLog.getAddress('MCD_JUG');
+        linkClipAddr = chainLog.getAddress('MCD_CLIP_LINK_A');
+        linkPipAddr = chainLog.getAddress('PIP_LINK');
+    }
+
+    function setInterfaces() private {
+        hevm = Hevm(hevmAddr);
+        link = GemAbstract(linkAddr);
+        vat = VatAbstract(vatAddr);
+        dai = DaiAbstract(daiAddr);
+        linkJoin = GemJoinAbstract(linkJoinAddr);
+        dog = DogAbstract(dogAddr);
+        jug = JugAbstract(jugAddr);
+        linkClip = ClipAbstract(linkClipAddr);
+        linkPip = OsmAbstract(linkPipAddr);
+    }
+
+    VaultHolder          ali;
+    UniswapV3SplitCallee dan;
+
+    address aliAddr;
+    address danAddr;
+    address UniswapV3Router2;
+    bytes   uniswapTxDataProfit;
+    bytes   uniswapTxDataNoProfit;
+
+    function getPermissions() private {
+        hevm.store(dogAddr, keccak256(abi.encode(address(this), uint256(0))), bytes32(uint256(1)));
+        hevm.store(vatAddr, keccak256(abi.encode(address(this), uint256(0))), bytes32(uint256(1)));
+        hevm.store(linkPipAddr, keccak256(abi.encode(address(this), uint256(0))), bytes32(uint256(1)));
+        linkPip.kiss(address(this));
+    }
+
+    function setEnvVars() private {
+        // Execute this test with fresh data via
+        // `(cd scripts/uniswap-split-route-callee && npm ci && node index.js)`
+        // check the script for more details on how to use universal router
+        UniswapV3Router2 = hevm.envOr('UniswapV3Router2', 0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45);
```

Comment:
Should use the new constant as the right parameter, left param env var style should be same as `ONE_INCH_ROUTER`.

Reply:
mm I think you missed my suggestion to use `uniV3Router` constant instead of the explicit address here.

Reply:
addressed